ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -470,3 +470,8 @@ def tensor_diag_part(input, name=None):
     reshaped = ivy.reshape(input, (prod, prod))
     diagonal = ivy.diagonal(reshaped)
     return ivy.reshape(diagonal, tuple(half_shape))
+
+
+@to_ivy_arrays_and_back
+def lu_solve(lower_upper, rhs):  # ToDO
+    return ivy.solve(lower_upper=lower_upper, rhs=rhs)
```

Comment:
`ivy.solve` function does not have any keyword arguments `lower_upper` and `rhs` :
https://unify.ai/docs/ivy/docs/functional/ivy/linear_algebra/ivy.functional.ivy.linear_algebra.solve.html#solve

Reply:
@hello-fri-end 
There is no function on ivy documentation that matches tf.linalg.lu_solve that is why I was instructed by one of Ivy reviewers to implement it compositionally with a present ivy function. That is what I am trying to do here.  And that is why I added #Todo as a comment in the code.

Do you understand, sir?

Reply:
Hey @TomiwaPhilip, you are right that the function should be implemented compositionally - this means you should use existing ivy functions to replicate the behavior of the function you're implementing. However, it seems the way you have tried to implement this function doesn't match with the functionality of `tf.linalg.lu_solve` ( look at the source code here ; https://github.com/tensorflow/tensorflow/blob/v2.13.0/tensorflow/python/ops/linalg/linalg_impl.py#L937-L1032). Here's a blog post where you can understand how `tf.linalg.lu_solve` works -> https://biswajitsahoo1111.github.io/post/doing-linear-algebra-using-tensorflow-2/. Once you have understood what the native function is supposed to do, you can easily replicate the behavior using ivy functions in your implementation. Happy coding :)